utils.py

```diff
@@ -0,0 +1,81 @@
+import lxml.html
+from typing import TextIO, List
+import re
+
+__regexp_url__ = re.compile(r'(?i)\b((?:[a-z][\w-]+:(?:\/{1,3}|[a-z0-9%])|[a-z0-9.\-]+[.][a-z]{2,4}\/)'
+                            r'(?:[^\s\(\)<>]+|\((?:[^\s\(\)<>]+|(?:\([^\s\(\)<>]+\)))*\))+'
+                            r'(?:\((?:[^\s\(\)<>]+|(?:\([^\s\(\)<>]+\)))*\)|[^\s`!\(\)\[\]{};:\'".,<>?«»“”‘’]))')
+
+__regexp_domain__ = re.compile(r'([\w-]+(?:\.[\w-]+)+)')
+__regexp_dash_underline__ = re.compile(r'(-_+|_+-)')
+__regexp_6letter_word__ = re.compile(r'(?i)(?<![\w-])([a-z][\w-]{4}[a-z0-9])(?![\w-])')
```

Comment:
What's the point of naming variables/methods as in the case of class dunder methods?
Constants, for example, should be named in a caps lock case like `REGEXP_URL`